datadog_lambda/tracing.py

```diff
@@ -89,7 +90,39 @@ def _context_obj_to_headers(obj):
     }
 
 
-def extract_dd_trace_context(event):
+def extract_context_from_http_event(event):
+    headers = event.get("headers", {})
+    lowercase_headers = {k.lower(): v for k, v in headers.items()}
+
+    trace_id = lowercase_headers.get(TraceHeader.TRACE_ID)
+    parent_id = lowercase_headers.get(TraceHeader.PARENT_ID)
+    sampling_priority = lowercase_headers.get(TraceHeader.SAMPLING_PRIORITY)
+
+    return trace_id, parent_id, sampling_priority
+
+
+def extract_context_from_sqs_event(event):
+    first_record = event["Records"][0]
```

Comment:
Any step here might fail because the event object can be in any arbitrary format, should we add a try/catch around this, and have the ability to return an empty context?

Reply:
Oh, whoops, didn't see Tian's review. Looks like he already mentioned this